<launch>
    <!--<arg name="machine" default="None" doc="Name of the machine to remotely run the training"/>-->
    <arg name="checkpoint_dir"/>

    <group ns="cnn_training">
        <!-- run local -->
        <node  pkg="cnn_lanefollowing" type="train_node.py" name="train_node" output="screen" clear_params="false" required="true">
            <param name="test_dir" value="/home/dominik/workspace/duckietown_imitation_learning/test_images"/>
            <param name="train_dir" value="/home/dominik/workspace/duckietown_imitation_learning/train_images"/>
            <param name="checkpoint_dir" value="$(arg checkpoint_dir)"/>
        </node>
    </group>
</launch>

